deployment-aws/terraform/cloudtrail.tf

```diff
@@ -1,11 +1,13 @@
-resource "aws_cloudtrail" "webmev-cloudtrail" {
-  name                    = "${local.common_tags.Name}"
-  s3_bucket_name          = aws_s3_bucket.log_bucket.id
-  s3_key_prefix           = "${local.stack}"
-  include_global_service_events = false
+resource "aws_cloudtrail" "default" {
```

Comment:
Sorry to nitpick but the indentation is still off. There are plugins for PyCharm and VSCode that do it automatically. Also, there's `terraform fmt [target]` command.